analyzer/src/main/kotlin/managers/utils/MavenSupport.kt

```diff
@@ -218,13 +218,14 @@ class MavenSupport(workspaceReader: WorkspaceReader) {
 
         fun parseVcsInfo(mavenProject: MavenProject): VcsInfo {
             // When asking Maven for the SCM URL of a POM that does not itself define an SCM URL, Maven returns the SCM
-            // URL of the parent POM (if any) and appends the child POM's artifactId to it. This behavior is
-            // fundamentally broken because it invalidates the URL for many SCMs that cannot clone / checkout a specific
-            // path from a repository. Also, the assumption that the source code for a child artifact is stored in a
-            // top-level directory named like the artifactId inside the parent artifact's repository is often not
-            // correct.
-            // To fix this, determine the SCM URL of the root parent (if there are parents) and use that as the child's
-            // SCM URL.
+            // URL of the first parent POM (if any) that deines one and appends the artifactIds of all child POMs to it,
```

Comment:
As there is just one parent, how about using the term `ancestor` instead? (same again in line 227) e.g.: "Maven return the SCM URL of the closest ancestor that declares one.." 

type: deines ->  defines / declares

Reply:
I think it's better to stay with "parent" because that's the term Maven uses.

Reply:
Maybe [this](https://maven.apache.org/plugins-archives/maven-dependency-plugin-2.9/display-ancestors-mojo.html) counts as Maven terminology as well?

Reply:
I also prefer to stick with "parent" as that's [the name of the XML tag](http://maven.apache.org/pom.html#Inheritance).